src/patternfly/components/Dropdown/docs/code.md

```diff
@@ -26,13 +27,16 @@ The dropdown menu can contain either links or buttons, depending on the expected
 | `.pf-c-dropdown__toggle` | `<button>` | Defines the dropdown toggle |
 | `.pf-c-dropdown__toggle-icon` | `<i>` | Defines the dropdown toggle icon |
 | `.pf-c-dropdown__toggle-text` | `<span>` | Defines the dropdown toggle text |
+| `.pf-c-dropdown__toggle-check` | `<label>` | Defines a checkbox in the toggle area of a split button dropdown |
```

Comment:
Can `.pf-c-dropdown__toggle-check` be used without `.pf-c-dropdown__toggle-button`? Or is it required that you use the button?

Reply:
A split button can take any type of interactive element. So far the examples I've seen are 2 or 3 `.pf-c-dropdown__toggle-button`'s or something like the example we have now with a `.pf-c-dropdown__toggle-check` and a `.pf-c-dropdown__toggle-button`

Reply:
ok great!